### PR TITLE
feat: Create 2022-07-26-aggiornato-catalogo-software-p

### PR DESCRIPTION
**Pull request by Francescopog**
## Description

This PR tackles:
@bfabio 
* ...
* ...
* ...

In particular, the ...

## Checklist

* [ ] I followed the indications in [CONTRIBUTING.md](https://github.com/italia/developers.italia.it/blob/master/CONTRIBUTING.md)
* [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] Ready for review! :rocket:

## Fixes

Fixes #

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>There is a problem with the commit message
> 2022-07-26-aggiornato-catalogo-software-pa.md
- subject may not be empty
- type may not be empty</td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 0d16358962aef30af80d262c96b7428bbc6f6532
</p>

File: _posts/it/2022-07-26-aggiornato-catalogo-software-pa.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- Università
+- PA centrali
+- Province
+- Istituti di ricerca e AFAM
+- Altri enti
+
+## L’aggiornamento del catalogo
+
+Developers Italia, in quanto progetto del Dipartimento per la trasformazione digitale e AGID, contribuisce, infatti, alla **Missione 1, Componente 1 del PNRR**, che ha l’obiettivo di trasformare in profondità la Pubblica Amministrazione attraverso una strategia centrata sulla digitalizzazione. 
+Nella ricerca di soluzioni che possano concorrere a raggiungere questo obiettivo, va rispettato quanto previsto dal Codice dell’Amministrazione Digitale (Artt. [68](https://docs.italia.it/italia/piano-triennale-ict/codice-amministrazione-digitale-docs/it/v2021-07-30/_rst/capo_VI-articolo_68.html) e [69](https://docs.italia.it/italia/piano-triennale-ict/codice-amministrazione-digitale-docs/it/v2021-07-30/_rst/capo_VI-articolo_69.html)), effettuando una **valutazione comparativa** che deve valutare i software disponibili secondo criteri tecnico economici, dando priorità a quelli a riuso od open source (che si possono trovare nel catalogo di Developers Italia) e motivando una loro eventuale esclusione a favore di soluzione proprietarie.
```
Comment (ruphy) [inline _posts/it/2022-07-26-aggiornato-catalogo-software-pa.md:46]:
```suggestion
Nella ricerca di soluzioni che possano concorrere a raggiungere questo obiettivo, va rispettato quanto previsto dal Codice dell’Amministrazione Digitale (Artt. [68](https://docs.italia.it/italia/piano-triennale-ict/codice-amministrazione-digitale-docs/it/v2021-07-30/_rst/capo_VI-articolo_68.html) e [69](https://docs.italia.it/italia/piano-triennale-ict/codice-amministrazione-digitale-docs/it/v2021-07-30/_rst/capo_VI-articolo_69.html)), effettuando una **valutazione comparativa** che deve valutare i software disponibili secondo criteri tecnico economici, dando priorità a quelli a riuso od open source (che si possono trovare nel catalogo di Developers Italia) e motivando una loro eventuale esclusione a favore di soluzioni proprietarie.
```

File: _posts/it/2022-07-26-aggiornato-catalogo-software-pa.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+Per agevolare questo compito all'interno dei progetti finanziati dal PNRR, Developers Italia ha individuato **alcune informazioni che possono essere inserite nel file publiccode.yml**, su base volontaria e autocertificata. Il file è utilizzato per la pubblicazione delle [schede di presentazione](https://developers.italia.it/it/search?type=software_reuse&sort_by=release_date&page=0) del [catalogo del riuso](https://developers.italia.it/it/software).
```
Comment (ruphy) [inline _posts/it/2022-07-26-aggiornato-catalogo-software-pa.md:48]:
```suggestion
Per agevolare questo compito all'interno dei progetti finanziati dal PNRR, Developers Italia ha individuato **alcune informazioni che possono essere inserite nel catalogo grazie al file descrittivo [publiccode.yml](https://yml.publiccode.tools)**, su base volontaria e autocertificata. Il file è utilizzato per la pubblicazione delle [schede di presentazione](https://developers.italia.it/it/search?type=software_reuse&sort_by=release_date&page=0) del [catalogo del riuso](https://developers.italia.it/it/software).
```

File: _posts/it/2022-07-26-aggiornato-catalogo-software-pa.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (bfabio) [inline _posts/it/2022-07-26-aggiornato-catalogo-software-pa.md:81]:
```suggestion
```

File: _posts/it/2022-07-26-aggiornato-catalogo-software-pa.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "PNRR: aggiornato il catalogo del software della PA"
+subtitle: "Nuove opportunità per evidenziare e trovare soluzioni software in riuso utili al rispetto dei requisiti degli avvisi di PA digitale 2026."
+date: 2022-07-26 08:00:00 +0200
```
Comment (bfabio) [inline _posts/it/2022-07-26-aggiornato-catalogo-software-pa.md:5]:
```suggestion
date: 2022-07-27 08:00:00 +0200
```

**Review by bfabio — APPROVED** [review]

**Pull request merged**